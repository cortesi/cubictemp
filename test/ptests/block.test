<!--(block one)
    one
(end)-->
@!one!@

<!--(block two)-->
    two
<!--(end)-->
@!two!@

<!--(block three)-->
    @!bar!@
    @!two!@
<!--(end)-->
@!three!@

<!--(block four)-->
    @!bar!@
    @!hoo!@
<!--(end)-->
@!four(dict(bar="hoink", hoo="farble"))!@

<!--(block foo)-->
    @!bar!@
<!--(end)-->
@!foo(dict(bar="hoink"))!@
