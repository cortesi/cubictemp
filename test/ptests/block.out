one

two

rab
two

hoink
farble

hoink
